variables.tf

```
@@ -167,6 +167,17 @@ variable "hive_config_properties" {
   default     = [""]
 }
 
+variable "trino_memory_connector" {
+  type = object({
+    use_memory_connector = bool
+    max_data_per_node = string
+  })
+  default = {
+    use_memory_connector = true,
```

This should not be default enabled
```suggestion
    use_memory_connector = false,
```